internal/metrics/kms_decrypter.go

```diff
@@ -25,25 +25,29 @@ type (
 	}
 
 	kmsDecrypter struct {
-		kmsClient *kms.KMS
+		kmsClient *kms.Client
+	}
+
+	clientDecrypter interface {
+		Decrypt(context.Context, *kms.DecryptInput, ...func(*kms.Options)) (*kms.DecryptOutput, error)
```

Comment:
Does AWS not provide the KMSAPI interface anymore?

Reply:
Seems like not with v2 😭